src/ui/Breadcrumb/Breadcrumb.ts

```diff
@@ -167,6 +168,15 @@ export class Breadcrumb extends Component {
       (firstEnabledResultList.element as any).focus({ preventScroll: true });
     }
   }
+
+  private addDefaultAccessibilityAttributes() {
```

Comment:
Curious as to why you have checking if there is not and attribute already. Is it because other more specific breadcrumbs already have those defined? Is it possible to make sure the default is set before the others?

Reply:
This is to allow clients to choose a different role or label, as there's a chance they may not want to use the `navigation` role, for instance.